client/dist/bundle.js.LICENSE.txt

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
I think you can use git restore --staged <filename> to remove this file from the staging area/commit. This file is compiled each time and so there's no need to push it up w/ your commit